gen/dcos-config.yaml

```diff
@@ -3002,3 +3002,60 @@ package:
           Port 61092
           Chunk_Size 32
           Buffer_Size 64
+  - path: /etc/dcos/network/cni/calico.conf
```

Comment:
Collides with https://github.com/mesosphere/dcos-enterprise/pull/7006/files#diff-8acc8e77a15678f67a353d405fcf971cR1647

Reply:
We had an examples to define files in both gen/dcos-config.yaml and gen_extra/dcos-config.yaml
/etc_slave/telegraf/telegraf.d/agent.conf in [OSS](https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L2623) amd [EE](https://github.com/mesosphere/dcos-enterprise/blob/master/gen_extra/dcos-config.yaml#L1284)
and 
/etc/dcos-ui-update-service.env in [OSS](https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L2925) and [EE](https://github.com/mesosphere/dcos-enterprise/blob/master/gen_extra/dcos-config.yaml#L1638)

Let me double-check this.

Reply:
The behavior is confirmed. This is tricky.
In case we have the same file defined in both ` gen/dcos-config.yaml` and ` gen_extra/dcos-config.yaml`, these paths will be arranged in a list, and as the content of `gen_extra/dcos-config.yaml` is added later, so the file defined in `gen_extra/dcos-config.yaml` will show up later than the one defined in  `gen/dcos-config.yaml`, hence, when iterating the file path list , the later one will overwrite the first one.

Reply:
Here is an example:
gen/dcos-config.yaml
```
package:
  - path: /bin/detect_ip
    permissions: "0755"
```
gen_extra/dcos_config.yaml
```
package:
  - path: /bin/detect_ip
    permissions: "0777"
```

After rendering the templates, we'll have 
```
{'dcos-config.yaml': {'package': [{'path': '/bin/detect_ip', 'permissions': '0755'}, {'path': '/bin/detect_ip', 'permissions': '0777'}]}}
```

Reply:
😲 Quite surprising.

Reply:
Thanks for double-checking it @mainred !